oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/source/DefaultScopeDefine.java

```diff
@@ -309,6 +316,16 @@ public static boolean inEndpointRelationCatalog(int scopeId) {
         return ENDPOINT_RELATION_CATALOG.containsKey(scopeId);
     }
 
+    /**
+     * Check whether current service belongs process catalog
```

Comment:
```suggestion
     * Check whether the given scope ID belongs process catalog
```
I think there are several similar typos in `in*Catalog` methods, please fix them too.

Reply:
Fixed.